examples/prepare-config-toml.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/sh
+set -eu
+
+config_toml="$(mktemp)"
+
+snapshotter="$1"
+pool_name="$2"
+
+case "$snapshotter" in
+    'naive')
+        cat > "$config_toml" <<EOF
+[proxy_plugins]
+  [proxy_plugins.firecracker-naive]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/naive-snapshotter.sock"
```

Comment:
It might be somewhat easier to use config imports here.
https://github.com/containerd/containerd/pull/3310#issuecomment-504182696
https://github.com/containerd/containerd/pull/3574

Reply:
Nice! Then I think we can have imports = ["/etc/containerd/snapshotter/*.toml"] and bind-mount the files.

Reply:
Seems import needs containerd upgrade #302

Reply:
import is now working!